docs/self-hosted-appcircle/install-server/helm-chart/installation/docker-k8s-migration.md

```diff
@@ -0,0 +1,1343 @@
+      -----END CERTIFICATE-----
+# Authentication configuration
+auth:
+  auth-keycloak:
+    # Organization name for Appcircle server
+    organizationName: spacetech
+    # Initial admin user email for Appcircle server
+    initialUsername: "admin@example.com"
+    # Initial admin password - Should contain: min 6 chars, 1 lowercase, 1 uppercase, 1 number
+    # You can create a secret with the password or directly enter the password here
+    initialPassword: "superSecretAppcirclePassword1234"
+    image:
+      # Appcircle keycloak image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-keycloak
+
+# Internal Ingress controller configuration
+ingress-nginx:
+  enabled: false
+
+# Appcircle vault configuration
+vault:
+  server:
+    image:
+      # Appcircle vault image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-vault
+
+# Web event Redis configuration
+webeventredis:
+  # Enable TLS for Redis connections
+  tls:
+    enabled: true
+  # Ingress configuration for Redis
+  ingress:
+    enabled: true
+    tls: true
+```
+
+</details>
+
+  </TabItem>
+</Tabs>
+
+### 2. Remove Sensitive Information From `values.yaml`
+
+**Remove sensitive information** such as Appcircle initial user password, SMTP password, SSL certificates, and other secrets from the `values.yaml` **for production environments**, by checking the [Sensitive Values](/self-hosted-appcircle/install-server/helm-chart/configuration/sensitive-configuration) documentation.
+
+### 3. Prepare `values.yaml` for Migration
+
+**Update your `values.yaml` file** with the following configurations before deploying the Helm chart to ensure a smooth migration process.
+
+---
+
+#### **Storage Updates**
+
+After [checking the data size](#3-check-the-data-size-on-the-standalone-appcircle-server) on the standalone Appcircle server, configure the storage sizes in `values.yaml` to meet or exceed your current data size requirements. Adequate storage is essential for a successful migration and seamless operation of the Appcircle server.
+
+For detailed instructions, refer to the [storage configuration](/self-hosted-appcircle/install-server/helm-chart/configuration/storage-configuration) page.
+
+---
+
+#### **Keycloak Updates**
+
+Ensure the `organizationName` and `initialOrganizationId` in the `values.yaml` file match those in the standalone Appcircle server.
+
+To locate these values:
+1. Go to the standalone Appcircle server dashboard.
+2. Switch to the root organization.
+3. Open the developer tools and switch to the "Network" tab.
+4. Navigate to your **Organization** page to retrieve the details.
+5. You can directly see the organization name in the UI.
+6. For the organization ID, filter the `userinfo` request.
+7. In the response, check for `currentOrganizationId`.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4705-organization-page.png' />
```

Comment:
Remove UI steps with the command below for "organization ıd"
- `cat generated-secret.yaml | grep -i organization`

Organization will be got from UI but fewer steps.

Reply:
Organization name might be optional but we're not sure now. Testing...